tools/src/commands/GenerateBareApp.ts

```diff
@@ -20,6 +21,7 @@ export async function action(
     name: appName = 'my-generated-bare-app',
     outDir = 'bare-apps',
     template = 'expo-template-bare-minimum',
+    localTemplate = false,
```

Comment:
Hmm, I think it would be more clear if default values were specified in the `program.option(...)` calls. Then it afaik appears in `--help`.

Reply:
Hi